main.js

```diff
@@ -168,3 +168,30 @@ detailCardClose4.addEventListener('click', () => {
 });
 // Card 4 popup opening closing ends Here
 // Adding Dynamic Data to Works ul ends Here
+
+// Contact Form Validation Starts Here
+// This Function will show error message
+const contactForm = document.getElementById('contact-me-form');
+const contactFormError = document.getElementById('contact-me-form-error-msg');
+const emailRegex = /^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$/;
+
+contactForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  if (contactForm.elements.name.value.trim() === '') {
+    contactFormError.innerText = 'Please enter your name';
+    contactFormError.classList.add('contact-me-form-error-msg-display');
+  } else if (!emailRegex.test(contactForm.elements.email.value.trim())) {
+    contactFormError.innerText = 'Please enter your email in lower case';
+    contactFormError.classList.add('contact-me-form-error-msg-display');
+  } else if (contactForm.elements.message.value.trim() === '') {
+    contactFormError.innerText = 'Please enter your message';
+    contactFormError.classList.add('contact-me-form-error-msg-display');
+  } else {
+    contactFormError.innerText = '';
+    contactFormError.className = 'contact-me-form-error-msg-hidden';
+    contactForm.submit();
+  }
+});
```

Comment:
- [ ] Good job using Regex for your validation. However, as required for the project kindly ensure that  If a user submits the contact form with the email field containing `uppercase` characters an error message should show up. Right now, your application sends a contact form for both cases: email with `uppercase` and `lowercase` characters. Make sure you fix that.